foundation/foundation-configuration-store/src/test/resources/specifications/load-configurations.feature

```diff
@@ -16,4 +16,10 @@ Feature: Load configurations at specified URI based on environment context
   Scenario: The properties are not distinguishable
     Given URIs pointing to nondistinct configurations
     When the configurations are loaded
-    Then an exception is thrown stating configurations are not distinguishable
\ No newline at end of file
+    Then an exception is thrown stating configurations are not distinguishable
+
+  @config-service
+  Scenario: configuration service returns the configuration value
+      Given the configuration service starts
```

Comment:
A: Do we want to make this `...service has started` instead of `...service starts` to differentiate it from the when?